traffic_control/clients/python/to_access/__init__.py

```diff
@@ -0,0 +1,576 @@
+
+.. option:: --to_host HOST
+
+	The :abbr:`FQDN (Fully Qualified Domain Name)` and optionally the port and scheme of the Traffic
+	Ops server. This will override :envvar:`TO_URL`. The format is the same as for :envvar:`TO_URL`.
+	(Default: uses the value of :envvar:`TO_URL`)
+
+.. option:: --to_password PASSWORD
+
+	The password to use when authenticating to Traffic Ops. Overrides :envvar:`TO_PASSWORD`.
+	(Default: uses the value of :envvar:`TO_PASSWORD`)
+
+.. option:: --to_user USERNAME
+
+	The username to use when connecting to Traffic Ops. Overrides :envvar:`TO_USER`. (Default: uses
+	the value of :envvar:`TO_USER`)
+
+Environment Variables
+---------------------
+If defined, :program:`toaccess` scripts will use these environment variables to define their
+connection to and authentication with the Traffic Ops server. Typically, setting these is easier
+than using the long options :option:`--to_host`, :option:`--to_user`, and :option:`--to_password` on
+every invocation.
+
+.. envvar:: TO_PASSWORD
+
+	Will be used to authenticate the user defined by either :option:`--to_user` or :envvar:`TO_USER`.
+
+.. envvar:: TO_URL
+
+	The :abbr:`FQDN (Fully Qualified Domain Name)` of the Traffic Ops server to which the script
+	will connect. The format of this should be :file:`[{http or https}://]{hostname}[:{port}]`. Note
+	that this may optionally start with ``http://`` or ``https://`` (case insensitive), but
+	typically this is unnecessary. Also notice that the port number may be specified, though again
+	this isn't usually required. All :program:`toaccess` scripts will assume that port 443 should be
+	used unless otherwise specified. They will further assume that the protocol is HTTPS unless
+	:envvar:`TO_URL` (or :option:`--to_host`) starts with ``http://``, in which case the default port
+	will also be set to 80 unless otherwise specified in the URL.
+
+.. envvar:: TO_USER
+
+	The name of the user as whom to connect to the Traffic Ops server.
+
+Module Reference
+================
+
+"""
+import json
+import logging
+import os
+import sys
+import typing
+
+import requests
+
+from trafficops.restapi import LoginError, OperationError, InvalidJSONError
+from trafficops.tosession import TOSession
+
+l = logging.getLogger()
+l.disabled = True
+logging.basicConfig(level=logging.CRITICAL+1)
+
+#: The full path to a file used to store the user's Mojolicious athentication cookie (currently unused)
+COOKIEFILE = "" #os.path.expanduser(os.path.join("~", ".to-auth.cookie"))
+
+def setCookie(cookie:str):
+	"""
+	Writes the passed cookie to the :data:`COOKIEFILE` file for later use.
+
+	.. warning::
+		Currently this function is never used, and it depends on the value of :data:`COOKIEFILE`,
+		which at the moment is being set to an empty string.
+	"""
+	with open(COOKIEFILE, "w") as f:
+		f.write(cookie)
+
+def output(r:requests.Response, pretty:bool, reqHeader:bool, respHeader:bool, reqPayload:bool,
```

Comment:
Similar to my pep8 comment above, parameters/variables should generally be snake_case as well. Seeing camelCase stuff that isn't a class name in python is a little weird to me. We should just stick to the pep8 conventions since that's what most Python programmers are used to anyways.

Reply:
I think we should be consistent within a file, follow naming conventions that are distinct for distinct objects, and use the conventions set forth in our project's `.pylintrc` file.

PEP8 sets conventions for *contributions to the CPython codebase*, not *Python projects in general*.